database/tables/author.table.sql

```diff
@@ -0,0 +1,8 @@
+DROP TABLE IF EXISTS AUTHOR CASCADE;
+
+CREATE TABLE AUTHOR(
+   ID         SERIAL PRIMARY KEY      NOT NULL,
```

Comment:
You can check but I think there is no need to use NOT NULL for PRIMARY KEY, cause it's cannot be nullable